rnns/wsj.pkl

```diff
@@ -0,0 +1,2758 @@
+(lp0
```

Comment:
Is this checked in on purpose?

Reply:
Yes, but I can just read the file from `scripts/mini_sequence_labeler/wsj.pkl` instead

Reply:
No, that's fine. Just making sure it's not an accidental check in.